<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_Telecommande</class>
 <widget class="QWidget" name="f_Telecommande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LbNomTelecommande">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>230</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>TÉLÉCOMMANDE
 CONNECTÉ À LA BROCHE N° 25</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LbImageTelecommande">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LbBoutonTelecommande">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
